src/Details.js

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {theme} from '@primer/components'
+import {space, color} from 'styled-system'
+
+const DetailsReset = styled('details')`
```

Comment:
This is a custom Details component that mimics the specific behavior we want for the NavDropdowns. Need to give it a little more thought to decide if it should be rolled into Primer Components. It basically provides overlay functionality in a way that still allows users to click on items in the dropdown and closes the dropdown when another dropdown is opened which was not possible with the CSS implementation